# Contains 4 discrete values

int64 id 
int64 object_size 
int64 human_expression 
int64 human_action
